### PR TITLE
add smart contract security

### PR DESCRIPTION
**Pull request by saeidshirazi**
add A curated list of Smart Contract Security materials and resources For Researchers

### CODE REVIEW

**Comment by saeidshirazi** [comment]
add new curated awesome list

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

File: readme.md [hunk 1]
```diff
@@ -628,6 +628,8 @@
 
 ## Security
 
+- [Smart Contract Security](https://github.com/saeidshirazi/Awesome-Smart-Contract-Security)
+- [Awesome-Android-Security](https://github.com/saeidshirazi/awesome-android-security)
```
Comment (IAmCoder) [inline readme.md:632]:
```suggestion
- [Awesome-Android-Security](https://github.com/saeidshirazi/awesome-android-security#readme)
```

File: readme.md [hunk 2]
```diff
@@ -628,6 +628,8 @@
 
 ## Security
 
+- [Smart Contract Security](https://github.com/saeidshirazi/Awesome-Smart-Contract-Security)
```
Comment (IAmCoder) [inline readme.md:631]: The URL to should end in `#readme`:
```suggestion
- [Smart Contract Security](https://github.com/saeidshirazi/Awesome-Smart-Contract-Security#readme)
```

**Review by IAmCoder — CHANGES_REQUESTED** [review]

**Comment by DevilDevaa** [comment]
Vbnj

**Comment by DevilDevaa** [comment]
> add A curated list of Smart Contract Security materials and resources For Researchers
> 
>

**Pull request closed without merging**